### PR TITLE
Devel rpc refactor

### PR DESCRIPTION
**Pull request by nickfh7**
Simple refactor of handlers and clients, should make it easier to read/manage. Also includes outlines of all client calls so we can reference them for later dev work.

### CODE REVIEW

File: manager-rs/dauth-service/src/rpc/handlers/handler.rs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+use std::sync::Arc;
+
+use crate::data::context::DauthContext;
+
+/// Handles all RPC calls to the dAuth service.
+pub struct DauthHandler {
```
Comment (matt9j) [inline manager-rs/dauth-service/src/rpc/handlers/handler.rs:6]: What's up with the handlers::handler redirection? Will this DauthHandler (which is pretty empty now) be needed in the long run? It seems nicer and more consistent to me to have everything be at "handers::{NAME}", and this module sticks out as not following the pattern.

**Review by matt9j — CHANGES_REQUESTED** [review]
Seems much cleaner now! One clarification question about the handers::handler module, otherwise LGTM.

File: manager-rs/dauth-service/src/rpc/handlers/handler.rs (hunk 1 above)
Comment (nickfh7) [reply manager-rs/dauth-service/src/rpc/handlers/handler.rs:6]: For sure, I'll add separate handlers

Comment (nickfh7) [reply manager-rs/dauth-service/src/rpc/handlers/handler.rs]: Okay, added the handlers

**Review by matt9j — APPROVED** [review]

**Pull request merged**